maths/binary_to_Decimal_conversion

```diff
@@ -0,0 +1,25 @@
+/**
+ * @function binaryToDecimal
+ * @description Convert the  binary to decimal .
+ * @param {number} binary - The input string
+ * @return {string} -  decimal of binary.
+ * @example binaryToDecimal('1011') = 11
+ * @example binaryToDecimal('1110') = 14
+ */
+
+function binaryToDecimal(binary: string): number {
+  let decimal: number = 0;
+  let power: number = 0;
+  for (let i = binary.length - 1; i >= 0; i--) {
+    if (binary[i] === '1') {
+      decimal += Math.pow(2, power);
+    }
+    power++;
+  }
+  return decimal;
+}
+
+
+const binary: string = '1011';
+const decimal: number = binaryToDecimal(binary);
+console.log(`The decimal representation of ${binary} is ${decimal}`);
```

Comment:
Side effect. Use a proper Jest test instead.

Reply:
Please check my code and merge it.

Reply:
> Side effect. Use a proper Jest test instead.

@appgurueu please check my code and merge it.